walsamlee/politico
# #163416728 View-party Endpoint

#### What does this PR do?
Add `view-parties` API endpoint
#### Description of Task to be completed?
Have `/api/v1/parties` endpoint working
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-v1-view-parties-163416728`
- Using postman, `GET` with API endpoint URL `/api/v1/parties` 
#### What are the relevant pivotal tracker stories?
#163416728
